### PR TITLE
Remove `full` endpoint and related methods

### LINKED ISSUE #513: Remove unused `/full` payload consumer
Since the TrustyAI service only operates with partial payload, the `full` consumer endpoint is never used and should be removed.

### PR DESCRIPTION
**Pull request by ruivieira**

### CODE REVIEW

File: explainability-service/src/test/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpointFormatsTest.java [hunk 1]
```diff
@@ -54,121 +54,6 @@ class ConsumerEndpointFormatsTest {
     @Inject
     Instance<MockMemoryStorage> storage;
 
-    static Prediction generateSingleInputSingleOutputPrediction() {
```
Comment (RobGeada) [inline explainability-service/src/test/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpointFormatsTest.java:57]: Wondering why all of these methods are removed, do we not need to generate example MM payloads anymore?

Comment (RobGeada) [reply explainability-service/src/test/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpointFormatsTest.java:57]: or are these full-payload generators?

Comment (ruivieira) [reply explainability-service/src/test/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpointFormatsTest.java:57]: Those methods where used by the `*Full` test methods, not the `*Partial` ones.
But one area that needs to be re-implemented is the "no data" scenarios. Those test were done exclusively with the "full" enpoint, they should be done with the "partial" endpoint now. I'll create a ticket for this.

**Review by RobGeada — APPROVED** [review]

**Pull request merged**